src/components/Input/Input.tsx

```diff
@@ -55,25 +54,6 @@ function Input({ setTasks, toggleHandler }: InputProps) {
       if (event.key === "Escape") {
         toggleHandler(false);
       }
-      // if (event.key === "Enter") {
-      //   event.preventDefault();
-      //   setTouched(true);
-      //   if (enteredTask === "") {
-      //     setValid(false);
-      //   } else {
-      //     setValid(true);
-      //     const newtitle = enteredTask;
-      //     const newComplete = false;
-      //     const obj = {
-      //       id: uuidv4(),
-      //       title: newtitle,
-      //       complete: newComplete,
-      //     };
-      //     setTasks((tasks) => [...tasks, obj]);
-      //     localStorage.setItem("setdate", formattedDate.toString());
-      //     setEnteredTask("");
-      //   }
-      // }
     };
 
     document.addEventListener("keydown", handleKey);
```

Comment:
Since you've extracted logic into onSubmit event handler, update dependencies of this side effect.
I see some unnecessary dependencies.